portal-web/test/functional/com/liferay/portalweb/macros/Staging.macro

```diff
@@ -936,6 +936,30 @@ definition {
 		Navigator.openWithAppendToBaseURL(urlAppend = "group/${siteURLKey}/~/control_panel/manage?p_p_id=com_liferay_staging_processes_web_portlet_StagingProcessesPortlet");
 	}
 
+	@summary = "Default summary"
+	macro openThreeDotsMenuItem(rowEntry = null, menuItem = null) {
+		var browserType = PropsUtil.get("browser.type");
+
+		if (${browserType} == "safari") {
```

Comment:
hey @DaniSzimko why can't we reuse the lexicon calls? I've searched the codebase and nobody else is doing this. If the lexicon utils are wrong, should we fix them instead of doing this?

Reply:
I have checked there are two macros in the `LexiconEntry.macro` with the same code but apparently different functionality:

```java
@summary = "Default summary"
macro gotoEntryMenuItem(rowEntry = null, menuItem = null) {
	LexiconEntry.openEntryMenu(rowEntry = ${rowEntry});

	MenuItem.click(menuItem = ${menuItem});
}
```

```java
@summary = "Default summary"
macro gotoEntryMenuItemNoWait(rowEntry = null, menuItem = null) {
	LexiconEntry.openEntryMenu(rowEntry = ${rowEntry});

	MenuItem.click(menuItem = ${menuItem});
}
```

Apparently, the first one should wait for the menu item and the second should not :thinking:

Reply:
@4lejandrito the LecixonEntry.macro's gotoEntryMenuItem() is used in multiple test cases. I created a new macro, because I did not want to change the existing behavior in other cases.